x/concentrated-liquidity/lp_test.go

```diff
@@ -551,6 +551,95 @@ func (s *KeeperTestSuite) TestIsInitialPosition() {
 	}
 }
 
+func (s *KeeperTestSuite) TestUpdatePosition() {
+	type updatePositionTest struct {
+		poolId          uint64
+		owner           sdk.AccAddress
+		lowerTick       int64
+		upperTick       int64
+		liquidityDelta  sdk.Dec
+		amount0Expected sdk.Int
+		amount1Expected sdk.Int
+		expectedError   bool
+	}
+
+	tests := map[string]updatePositionTest{
+		"happy path": {
+			poolId:          1,
+			owner:           s.TestAccs[0],
+			lowerTick:       DefaultLowerTick,
+			upperTick:       DefaultUpperTick,
+			liquidityDelta:  DefaultLiquidityAmt,
+			amount0Expected: DefaultAmt0Expected,
+			amount1Expected: DefaultAmt1Expected,
+			expectedError:   false,
+		},
+		"error: invalid pool id": {
+			poolId:          2,
+			owner:           s.TestAccs[0],
+			lowerTick:       DefaultLowerTick,
+			upperTick:       DefaultUpperTick,
+			liquidityDelta:  DefaultLiquidityAmt,
+			amount0Expected: DefaultAmt0Expected,
+			amount1Expected: DefaultAmt1Expected,
+			expectedError:   true,
+		},
+		"new position when calling update position": {
+			poolId:          1,
+			owner:           s.TestAccs[1],
+			lowerTick:       DefaultLowerTick,
+			upperTick:       DefaultUpperTick,
+			liquidityDelta:  DefaultLiquidityAmt,
+			amount0Expected: DefaultAmt0Expected,
+			amount1Expected: DefaultAmt1Expected,
+			expectedError:   false,
+		},
```

Comment:
This is a new position from a new owner. Do you think we should have a test case that calls update position from the same owner that originally created the position?

Reply:
if I understood you correctly, that's what the "happy path" test case does! Would update the test case name to make this clearer if I did understand you correctly